environment.py

```diff
@@ -912,5 +912,18 @@ def env() -> Mapping[str, Optional[str]]:
         # in the Azul VPC. This subnet can't overlap the VPC CIDR and the subnet
         # mask must be less than 22 bits.
         #
-        'azul_vpn_subnet': None
+        'azul_vpn_subnet': None,
+
+        # A global rate limit on file downloads across all regions and IP
+        # addresses, enforced by AWS WAF. The syntax is
+        # "<limit>/<window>@<concurrency>", where <limit> is the maximum allowed
+        # number of download requests made every <window> seconds, and
+        # <concurrency> is the expected number of distinct IPs making at least
+        # one download request during that time. The concurrency does not need
+        # to be an integer. See
+        # https://docs.aws.amazon.com/waf/latest/developerguide/waf-rule-statement-type-rate-based-high-level-settings.html # noqa E501
```

Comment:
```suggestion
        #
        # https://docs.aws.amazon.com/waf/latest/developerguide/waf-rule-statement-type-rate-based-high-level-settings.html # noqa E501
        #
```

I assume the `noqa` is actually needed. Otherwise, please remove it.